### DISCUSSION TITLE
Setup for practise exam

### DISCUSSION THREAD

**Discussion by Koen-Veldman** [tudelft/AE4353-Y24#31] [Exam]
Does the TU Delft Spyder setup on Brightspace already include all the necessary packages, or are we expected to install them manually before starting? I tried running the code for Question 10a, but it seems the required dependencies are missing.

**Comment by yilun-wu** [comment]
The installer should have included everything to run the code for the exam. No, you don't need to install anything on the exam computers - the conda environment will be there for you when you start.

Which dependencies are missing for you? Did you switch to the right conda environment called "pytorch"?

**Reply by mgutierreztudelft** [reply]
I am using the environment called tudelft/pytorch and I get the same problem - when executing the exercise 10a file I get the message ModuleNotFoundError: No module named 'torch

**Reply by yilun-wu** [reply]
It's likely that you haven't really switched on the pytorch environment. Try executing:
```
import sys
print(sys.executable)
```
in the python console. Do you see the name "pytorch" in the path?

**Reply by mgutierreztudelft** [reply]
Yes, I attach you a picture of it. It seems like the environment is set up.
![image](https://github.com/user-attachments/assets/4b3082af-9467-4260-bd71-eef58c261e11)

**Reply by yilun-wu** [reply]
Did you import torch before calling torch.manual_seed(0)?

**Reply by DidierVDVoort** [reply]
I had the same issue but a Spyder restart fixed the problem for me
